rllib/scripts.py

```diff
@@ -91,11 +92,14 @@ def run(example_id: str = typer.Argument(..., help="Example ID to run.")):
     if example_id not in EXAMPLES.keys():
         raise example_error(example_id)
 
-    example_file = EXAMPLES[example_id]["file"]
+    example = EXAMPLES[example_id]
+    example_file = example.get("file")
```

Comment:
What if `example_file` is None here? Do we have an error for this case?

Reply:
the existence of `example_id` is guaranteed by Typer validation (it's a mandatory arg). whether or not the `file` key exists in the dictionary in `common.py` is something we as a team have to ensure. Having said that, I think this is a good point. I'll add a quick error there.